dist/index.js

```diff
@@ -1271,7 +1271,7 @@ function run() {
             let versionSpec = core.getInput('ruby-version');
             if (!versionSpec) {
                 // deprecated
-                versionSpec = core.getInput('version');
+                versionSpec = core.getInput('version', { required: true });
```

Comment:
This might do the opposite of what we want, (Cause more users to set `version` when they get this error making it more painful to deprecate in the future)

Might just wanna throw a custom error

Reply:
yep - going to revert.  the yaml has a default value.

Reply:
fixed